zerver/lib/events.py

```diff
@@ -738,6 +738,9 @@ def _draft_update_action(i: int) -> None:
 
         if event["op"] == "add":
             person = copy.deepcopy(person)
+            if person["email_address_visibility"] != UserProfile.EMAIL_ADDRESS_VISIBILITY_EVERYONE:
+                client_gravatar = False
```

Comment:
Can't we check this one by checking whether `delivery_email` is None, without needing to provide the `email_address_visibility` field in the event?